packages/design-system/src/styles/components/_FilterChip.scss

```diff
@@ -16,14 +11,14 @@ $filter-chip-icon-size: 10px;
   margin-top: $spacer-1;
   min-height: 0;
   padding: $spacer-half $spacer-1;
-  padding-right: ($spacer-2 + $filter-chip-icon-container-size);
```

Comment:
It appears that this should be `34px`, and this discrepency is the cause for the visual regression reference changes for filter chip.